spec/api/v4/cvd_risk_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "swagger_helper"
+
+describe "CVD Risk V4 API", swagger_doc: "v4/swagger.json" do
+  let(:request_user) { create(:user) }
+  let(:request_facility) { create(:facility, facility_group: request_user.facility.facility_group) }
+  let(:HTTP_X_USER_ID) { request_user.id }
+  let(:HTTP_X_FACILITY_ID) { request_facility.id }
+  let(:Authorization) { "Bearer #{request_user.access_token}" }
+
+  path "/cvd_risks/sync" do
+    post "Syncs cvd_risks data from device to server." do
+      tags "CVD Risk"
+      security [access_token: [], user_id: [], facility_id: []]
+      parameter name: "HTTP_X_USER_ID", in: :header, type: :uuid
+      parameter name: "HTTP_X_FACILITY_ID", in: :header, type: :uuid
+      parameter name: :cvd_risks, in: :body, schema: Api::V4::Schema.cvd_risk_sync_from_user_request
+
+      response "200", "cvd_risks created" do
+        let(:cvd_risks) { {cvd_risks: (1..3).map { build(:cvd_risk).attributes.with_payload_keys }} }
+        run_test!
+      end
+
+      response "200", "some, or no errors were found" do
+        schema Api::V4::Schema.sync_from_user_errors
+        let(:cvd_risks) { {cvd_risks: (1..3).map { build(:cvd_risk, :invalid).attributes.with_payload_keys }} }
+        run_test!
+      end
+
+      include_examples "returns 403 for post requests for forbidden users", :cvd_risks
+    end
+
+    get "Sync patient attribute data from server to device" do
```

Comment:
This should say "Sync CVD risk data from server to device".

<img width="1233" alt="image" src="https://github.com/user-attachments/assets/7e2c19aa-7502-4de1-8e80-c3fb0f5e3eea">